pkg/service/schema.go

```diff
@@ -55,6 +55,7 @@ func (i InputParametersSchema) MarshalJSON() ([]byte, error) {
 				Schema: jsonSchemaVersion,
 				Type:   "object",
 				inputParametersSchema: inputParametersSchema(i),
+				Additional:            false,
```

Comment:
Always explicitly say that the top-level schema doesn't support additional properties. This is a critical in preventing attempts to update params that are in the provisioning params schema, but not in the updating params schema.